gradle-conjure/src/main/java/com/palantir/gradle/conjure/OsUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.conjure;
+
+import java.io.File;
+import org.apache.commons.lang3.SystemUtils;
```

Comment:
Isn't there a gradle-native way of checking this without requiring a new apache dependency?

Reply:
Looks like there's an internal gradle API to find this, but it has changed at least twice. There's a gradle-bundled ant thing I use instead with no extra deps.